TypeScriptReact.YAML-tmLanguage

```diff
@@ -46,13 +46,16 @@ repository:
      - include: '#enum-declaration'
 
   type-declaration:
-    name: meta.type.declaration.tsx
-    begin: \b(type)\b\s+([a-zA-Z_$][\w$]*)\s*=\s*
+    name: meta.type.declaration.ts
```

Comment:
All grammar defined in TypeScriptReact need to have a top level scope of `.tsx`
This change is unneeded.